server/hotrod/src/main/java/org/infinispan/server/hotrod/tx/table/PerCacheTxTable.java

```diff
@@ -20,6 +20,7 @@
  * @author Pedro Ruivo
  * @since 9.4
  */
+//TODO merge with org.infinispan.server.hotrod.tx.table.GlobalTxTable
```

Comment:
tracking issue ?

Reply:
It is already merged :) I'll open a PR later this week with this (and other) fixes.